vbr/vbrCore/functions/yosh2009.m

```diff
@@ -0,0 +1,66 @@
+function [ VBR ] = yosh2009( VBR )
```

Comment:
Oh -- one more suggestion while you're updating all these functions, I'd prefer if they all had a common prefix. Maybe `cond_yosh2009.m`, `cond_wang2006.m`, etc., to help organize the `functions` folder. This change can wait until after things are initially working...